opus/application/static_media/js/hash.js

```diff
@@ -45,6 +46,67 @@ var o_hash = {
         return hash.join("&");
     },
 
+    encodeSlugValue: function(slugValueArray) {
+        /**
+         * Take in a slug value array (like opus.selections, each element
+         * will be a value for the slug) and encode all values in the
+         * array. Return an array that contains encoded values for the
+         * slug. This function will be called in updateHash to make sure
+         * slug values in the hash are all encoded before updating the URL.
+         */
+        let slugValue = [];
+        $.each(slugValueArray, function(index, val) {
```

Comment:
Given ES6, Debby and I talked about the use of $.each. It's really not needed for such a simple case and has excess overhead. It might be more readable to use the new "for...of" construct:
`for (const val of slugValueArray) {`